db/migrate/20221019161001_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] it would be better if you change `:user` to `author` as asked in the requirements  and try to do the same as I commented in the above comment